meshtastic/interfaces/ble/receive_service.py

```
@@ -607,6 +610,65 @@ def _should_poll_without_notify(self) -> bool:
+            ) -> bool:
+                result = injected_wait_for_event(
+                    target_coordinator,
+                    event_name,
+                    timeout,
+                )
+                if isinstance(result, bool):
+                    return result
+                raise TypeError(
+                    "injected_wait_for_event must return bool in "
+                    "_validated_wait_for_runtime_event"
+                )
```

Ruff (0.15.6)

[warning] Avoid specifying long messages outside the exception class

(TRY003)